### PR TITLE
Schedule updated with Professor's updates

### PR DESCRIPTION
**Pull request by ShanTulshi**

### CODE REVIEW

File: _data/schedule.yml [hunk 1]
```diff
@@ -1,164 +1,154 @@
...
   materials: "LaValle, Ch 2.3, 3"
-  title: "[Bird's-eye view: Sensation and perception](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/lecture3.pptx)"
+  title: "[3D Computer Graphics: Geometric Modeling](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/lecture3.pptx)"
 
 - 
-  date: "Sep 10"
+  date: "Jan 28"
   materials: "LaValle, Ch 3.2, 3.3"
-  title: "[Geometric modeling, linear transformations, 2D rigid-body transformations](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture4.pptx)"
+  title: "[Design Processes – VR Design (General Considerations)](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture4.pptx)"
```
Comment (andrewipark) [inline _data/schedule.yml]: what about "Design Processes - General VR considerations" or similar?

File: _data/schedule.yml [hunk 2]
```diff
@@ -1,164 +1,154 @@
...
   materials: "LaValle, Ch 3.2, 3.3"
-  title: "[3D rotations, yaw, pitch, roll, quaternions, conversions](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture5.pptx)"
+  title: "[Mathematics for Orientation](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture5.pptx)"
 
 - 
-  date: "Sep 17"
+  date: "Feb 4"
   materials: "LaValle, Ch 3.2, 3.3"
-  title: "[Quaternions, conversion, Local vs Global coordinate frames. Homogeneous transforms](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture6.pptx)"
+  title: "Design Processes – VR Design (Ergonomics)"
```
Comment (andrewipark) [inline _data/schedule.yml]: "Design Processes - Ergonomics in VR"

File: _data/schedule.yml [hunk 3]
```diff
@@ -1,164 +1,154 @@
...
+  date: "Feb 6"
+  materials: ~
+  title: "**No Lecture: Exam Week**"
 
 - 
-  date: "Sep 24"
+  date: "Feb 11"
   materials: "LaValle, Ch 3"
-  title: "[Eye, canonical view, and viewport transforms](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture8.pptx)"
+  title: "Design Processes – VR Design (Input, Movement & User Interface)"
```
Comment (andrewipark) [inline _data/schedule.yml]: as above, saying "Design processes" and then "VR **design**" right after that feels redundant but I don't have a suggestion for this one

File: _data/schedule.yml [hunk 4]
```diff
@@ -1,164 +1,154 @@
...
   materials: ~
-  title: "No lecture - Thanksgiving Break"
+  title: "Design in more Detail (Mid-level processes)"
 
 - 
-  date: "Nov 26"
+  date: "Apr 17"
   materials: ~
-  title: ~
+  title: "Production in more Detail (Studios, Teams and lower level processes)"
```
Comment (andrewipark) [inline _data/schedule.yml]: I advise against parenthetical clauses. Especially for the first one, a possible shortened version is "Survey of VR in the Game Industry". Also for all of these, the title case should be consistently applied (or not).

File: _data/schedule.yml [hunk 5]
```diff
@@ -1,164 +1,154 @@
...
   materials: "LaValle, Ch 2.1, 2.2"
-  title: "[Bird's-eye view: VR hardware](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture2.ppt)"
+  title: "[Design Processes - Basic Principles and Top Down Design Processes](https://courses.engr.illinois.edu/cs498vr3/fa2018/lectures/Lecture2.ppt)"
 
 - 
-  date: "Sep 3"
+  date: "Jan 21"
   materials: ~
-  title: "No lecture"
+  title: "**No Lecture: M. L. King Day**"
```
Comment (andrewipark) [inline _data/schedule.yml]: There is enough room to put the full name "Martin Luther King Jr. Day". I have never seen this specific abbreviation before.

File: _data/schedule.yml (hunk 1 above)
Comment (shaffer1) [reply _data/schedule.yml]: Yeah - I'll change to colons instead of parentheses

File: _data/schedule.yml (hunk 4 above)
Comment (shaffer1) [reply _data/schedule.yml]: Good suggestion - thanks

File: _data/schedule.yml (hunk 5 above)
Comment (shaffer1) [reply _data/schedule.yml]: That's the wording on the official UIUC calendar that I copied. Happy to change it though.

**Comment by shaffer1** [comment]
I'm going to deploy and finalize changes in the master branch

**Pull request merged**